packages/core-v2-typechain/.prettierignore

```diff
@@ -0,0 +1,10 @@
+*.json
+*.js
+*.md
```

Comment:
don't think enabling prettier for json or md will cause any harm. will be good for readme.md